<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>527</width>
    <height>253</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Communicator</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>Observe.ico</normaloff>Observe.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-30</y>
      <width>531</width>
      <height>261</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="tabsClosable">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="tab">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QLineEdit" name="IP_input">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>50</y>
        <width>371</width>
        <height>32</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>IP</string>
      </property>
     </widget>
     <widget class="QPushButton" name="connect_button">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>10</y>
        <width>61</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>연결</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="status_label">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>110</y>
        <width>421</width>
        <height>91</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Noto Sans Mono CJK KR</family>
        <pointsize>48</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;연결 대기중...&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="user">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>10</y>
        <width>121</width>
        <height>32</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>USER</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>10</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>PWD</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="pwd">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>10</y>
        <width>121</width>
        <height>32</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="clearButtonEnabled">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
     <widget class="QLineEdit" name="file_path">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>451</width>
        <height>20</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="log_text">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>501</width>
        <height>131</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="backgroundVisible">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="to_back">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>61</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>뒤로</string>
      </property>
     </widget>
     <widget class="QPushButton" name="get_file_path">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>38</y>
        <width>41</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>527</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>user</tabstop>
  <tabstop>pwd</tabstop>
  <tabstop>IP_input</tabstop>
  <tabstop>connect_button</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>to_back</tabstop>
  <tabstop>file_path</tabstop>
  <tabstop>get_file_path</tabstop>
  <tabstop>log_text</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>to_back</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>back()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>44</x>
     <y>31</y>
    </hint>
    <hint type="destinationlabel">
     <x>134</x>
     <y>-11</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>connect_button</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>ipCheck()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>403</x>
     <y>65</y>
    </hint>
    <hint type="destinationlabel">
     <x>386</x>
     <y>89</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>get_file_path</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>filePath()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>407</x>
     <y>61</y>
    </hint>
    <hint type="destinationlabel">
     <x>338</x>
     <y>-7</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>ipCheck()</slot>
  <slot>back()</slot>
  <slot>filePath()</slot>
 </slots>
</ui>
